tests/integration/conftest.py

```diff
@@ -127,6 +127,11 @@ def application_plan(api, service, application_plan_params) -> ApplicationPlan:
     resource = service.app_plans.create(params=application_plan_params)
     yield resource
 
+@pytest.fixture(scope='module')
+def application_plans(api) -> ApplicationPlans:
```

Comment:
This fixture is for all application plans for all products/services. `application_plan` fixture is only for one `service`. You have merged 2 different test areas 1) application plans for one service and 2) application plans for all services. Please split them to 2 different modules. It means that you should add listing of all application plans for one service to `test_integration_application_plan.py`, consider of course change of name of this module. And 2) create CRUD test in new module where you use this fixture, new client from `threescale_api/client.py`.